test/fhirtypes/Range.test.ts

```diff
@@ -0,0 +1,112 @@
+import cloneDeep from 'lodash/cloneDeep';
+import { Range, validateRange } from '../../src/fhirtypes/Range';
+import { Quantity } from '../../src/fhirtypes/Quantity';
+import { InvalidRangeValueError } from '../../src/errors/InvalidRangeValueError';
+import { UnitMismatchError } from '../../src/errors/UnitMismatchError';
+import { CodeAndSystemMismatchError } from '../../src/errors/CodeAndSystemMismatchError';
+
+describe('Range', () => {
+  let small: Quantity;
+  let large: Quantity;
+
+  beforeAll(() => {
+    small = {
+      value: 2,
+      unit: 'UCUM#mm',
+      system: 'http://unitsofmeasure.org',
+      code: 'mm'
+    };
+    large = {
+      value: 25,
+      unit: 'UCUM#mm',
+      system: 'http://unitsofmeasure.org',
+      code: 'mm'
+    };
+  });
+
+  describe('#validateRange', () => {
+    it('should be valid when one bound or both bounds are undefined', () => {
+      const onlyLow: Range = {
+        low: large
+      };
+      const onlyHigh: Range = {
+        high: small
+      };
+      const neitherBound: Range = {};
+      expect(validateRange(onlyLow)).toBe(true);
+      expect(validateRange(onlyHigh)).toBe(true);
+      expect(validateRange(neitherBound)).toBe(true);
+    });
+    it('should require low bound value to be less than or equal to high bound value', () => {
```

Comment:
Can you put a blank line between `it` tests?  For me, that helps me to read them easier.